specification/ai/Azure.AI.Projects/deployments/routes.tsp

```diff
@@ -0,0 +1,45 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "./models.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+using Azure.Core.Foundations;
+
+namespace Azure.AI.Projects;
+
+alias DeploymentServiceTraits = SupportsClientRequestId &
+  NoRepeatableRequests &
+  NoConditionalRequests;
+
+alias DeploymentOperations = Azure.Core.ResourceOperations<DeploymentServiceTraits>;
+
+interface Deployments {
+  @doc("Get a deployed model.")
+  get is DeploymentOperations.ResourceRead<Deployment>;
```

Comment:
For /deployments, not clear to me if the "get" method can also get a connected deployment. Should we not also add include_connection_deployments to the "get" method?

Reply:
You would use the "full" deployment name, which includes the connection name to get deployments on connections.